### DISCUSSION TITLE
How to set shortcuts for NERDtree

### DISCUSSION THREAD

**Discussion by exaV** [JetBrains/ideavim#356] [Q&A]
I would like to use `l` to open/expand nodes in NERDTree instead of (or additionally to) the default key `o`.
Is that possible? and how would I configure it?

I tried searching online and combining what I found abound configuring plugins with the actions listed here
https://github.com/JetBrains/ideavim/wiki/NERDTree-support in various ways, but to no effect. Typing `l` always triggers Idea's search.
I tried (none of it worked)
```
let g:NERDTreeMapActivateNode="l"
:map l <Plug>(NERDTree-g:NERDTreeMapActivateNode)
:map l <Plug>(NERDTree-o)
```

Also is there a way to use only `/` to start searching in the project explorer, as opposed to any key that isn't mapped to something in NERDtree?

**Comment by allworldg** [comment]
Actually , the solution you give is very helpful for me . Thank you .

**Reply by captainxyw** [reply]
How do you reset default key binding for NERDTree? bro

**Reply by allworldg** [reply]
@captainxyw I just change Preview :
```
let g:NERDTreeMapPreview="<Tab>"
```

**Comment by KrtNinja** [comment]
```
set NERDTree

let g:NERDTreeMapActivateNode = 'l'
let g:NERDTreeMapJumpParent = 'h'
```

After restart IDE, it helps me
